inference-engine/src/preprocessing/ie_preprocess_gapi_kernels.cpp

```diff
@@ -885,6 +892,10 @@ struct linearScratchDesc {
                     outH * sizeof(alpha_t)     +
                     outH * sizeof(index_t) * 2 +
                      inW * sizeof(T) * lpi * chanNum;
+#ifdef HAVE_NEON
```

Comment:
I would reinvent this, at least add comment

Reply:
Reworked